Tests/SafeDIToolTests/SafeDIToolCodeGenerationErrorTests.swift

```diff
@@ -1733,6 +1733,7 @@ final class SafeDIToolCodeGenerationErrorTests: XCTestCase {
         tool.moduleInfoOutput = nil
         tool.dependentModuleInfoFilePath = nil
         tool.dependencyTreeOutput = nil
+        tool.dotFileOutput = nil
```

Comment:
not strictly speaking necessary but it is best to set every property on the tool before trying to run it to avoid spurious errors.